### DISCUSSION TITLE
Free version, without using Durable Objects

### DISCUSSION THREAD

**Discussion by niutech** [emadda/tabserve#3] [Ideas]
Durable Objects are available only in the [Cloudflare Workers Paid plan](https://developers.cloudflare.com/workers/platform/pricing/#durable-objects). Please provide a free version of TabServe which will work using the free plan, possibly allowing only a single subdomain.
Thanks!

**Comment by emadda** [comment]
Hello @niutech,

This could work, but may end up being more expensive than the $5/month paid plan for Durable Objects as the WebSocket from the Cloudflare Worker to Tabserve would need to be always be active.

Durable Objects have the WebSocket Hibernation API, so the WebSocket remains connected but your worker sleeps so you are only charged when you send or receive messages.
